MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -85,7 +85,7 @@ class RouteManeuverViewController: UIViewController {
         let stepProgress = routeProgress.currentLegProgress.currentStepProgress
         let distanceRemaining = stepProgress.distanceRemaining
         
-        distance = secondsRemaining > 5 ? distanceRemaining : nil
+        distance = distanceRemaining > 10 ? distanceRemaining : nil
```

Comment:
Is it OK that, depending on the locale, we’d be hiding at the 11-yard or 33-foot mark instead of a clean number like the 10-meter mark?